addons/stock_account/models/stock_move.py

```diff
@@ -139,6 +139,14 @@ def _is_dropshipped_returned(self):
         self.ensure_one()
         return self.location_id.usage == 'customer' and self.location_dest_id.usage == 'supplier'
 
+    def _is_production(self):
+        self.ensure_one()
+        return self.location_id.usage == 'production' and self.location_dest_id._should_be_valued()
+
+    def _is_production_consumed(self):
+        self.ensure_one()
+        return self.location_id._should_be_valued() and self.location_dest_id.usage == 'production'
+
```

Comment:
The location usage 'production' should be in mrp. It's a clean to do. So avoid this kind of hack and try to keep everything related to production in mrp